server/container/guice/guice-common/src/main/java/org/apache/james/PeriodicalHealthChecks.java

```diff
@@ -100,6 +96,19 @@ private void logUnhealthy(Result result) {
         }
     }
 
+    private void logDegraded(Result result) {
```

Comment:
it could be factorized with logUnhealthy.

Reply:
They have different log levels: WARN/ERROR for Degraded/Unhealthy. I did try but haven't come up with any "pretty" solution yet. Do you have suggestion?

Reply:
Ok, the log method with `n` arguments are not the easier to factorize.